z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -224,8 +223,25 @@ icic-services status
 ```
 - Login to the IBM Cloud Infrastructure Center web console and select Home > Environment Checker, click the Run Environment Checker button to confirm the cluster does not have any failed messages.
 
-If you meet any **not running** service or **failed** message, check the IBM Cloud Infrastructure Center [Troubleshooting](https://www.ibm.com/docs/en/cic/1.1.5?topic=troubleshooting) document to fix before running the ansible playbook.
+If you meet any **not running** service or **failed** message, check the IBM Cloud Infrastructure Center [Troubleshooting](https://www.ibm.com/docs/en/cic/1.1.6?topic=troubleshooting) document to fix before running the ansible playbook.
 
+9. Subnet DNS (Required for KVM)
+
+**Note**: This step is required for KVM and only necessary if you want to use your own or external bastion server resolvers your Nova servers, z/VM is optional.
```

Comment:
`only necessary if you want to use your own or external bastion server resolvers your Nova servers,`
I think this description is indigestible. Just delete it. 
This step is required for KVM , z/VM is optional.

Reply:
done.